core/assets/bundles/bundle.properties

```diff
@@ -520,6 +518,17 @@ rules.respawntime = Respawn Time:[LIGHT_GRAY] (sec)
 rules.wavespacing = Wave Spacing:[LIGHT_GRAY] (sec)
 rules.buildcostmultiplier = Build Cost Multiplier
 rules.buildspeedmultiplier = Build Speed Multiplier
+rules.waitForWaveToEnd = Begin counting to next wave after all enemies are killed
+rules.respawns = Number of maximum respawns per wave
+rules.limitedRespawns = Should respawns be limted?
```

Comment:
(refer to comment on line 522) I don't think you should have a button just for this. I would say it's simpler to have just a number input and assign 0 to unlimited.